pkg/apis/m3dboperator/v1/cluster.go

```diff
@@ -75,6 +78,14 @@ type ClusterSpec struct {
 	Resources Resources `json:"resources"`
 }
 
+// IsolationGroup defines the name of zone as well attributes for the zone configuration
+type IsolationGroup struct {
+	// Name
+	Name string `json:"name"`
+	// InstancesAmount defines the amount of instances
+	InstanceAmount int32 `json:"instanceAmount"`
```

Comment:
What's the intent behind this value?

Reply:
This would be the amount of instances per isolation group.

Reply:
ha ok, how about renaming to `NumInstances`?